ref_builder/cli.py

```diff
@@ -235,12 +248,19 @@ def isolate_create(
         click.echo(f"OTU {taxid} not found.", err=True)
         sys.exit(1)
 
+    isolate_name_ = None
+    if isolate_name is not None:
+        isolate_name_type, isolate_name_value = isolate_name
+        isolate_name_ = IsolateName(type=isolate_name_type, value=isolate_name_value)
+
     try:
         add_isolate(
             repo,
             otu_,
             accessions_,
             ignore_cache=ignore_cache,
+            ignore_name=ignore_name,
+            isolate_name=isolate_name_,
```

Comment:
This function signature is getting murky.
I would split `add_isolate` into separate functions for each use case.